class/keycloak.yml

```diff
@@ -9,3 +15,73 @@ parameters:
           - keycloak/component/main.jsonnet
         input_type: jsonnet
         output_path: keycloak/
+      - output_path: keycloak/01_keycloak_helmchart
+        input_type: helm
+        output_type: yaml
+        input_paths:
+          - keycloak/helmcharts/keycloak
+        helm_values:
```

Comment:
Is there a reason we don't expose the values directly in the hierarchy?

Reply:
Addressed in #3 .